nginx-agent-docker/Dockerfile

```
@@ -0,0 +1,59 @@
+FROM debian:bullseye-slim
+
+ARG NMS_URL
+ARG DEVPORTAL=false
+ARG NAP_WAF=false
+
+# Initial packages setup
+RUN	apt-get -y update \
+	&& apt-get -y install -y apt-transport-https lsb-release ca-certificates wget gnupg2 wget curl \
+# NGINX Instance Manager agent setup
+	&& mkdir -p /deployment /etc/ssl/nginx \
+# Agent installation
+	&& bash -c 'curl -k $NMS_URL/install/nginx-agent | sh' && echo "Agent installed from NMS"
+
+# Startup script
+COPY ./container/start.sh /deployment/
+
+# Download certificate and key from the customer portal (https://account.f5.com)
+# and copy to the build context
+RUN --mount=type=secret,id=nginx-crt,dst=/etc/ssl/nginx/nginx-repo.crt,mode=0644 \
+	--mount=type=secret,id=nginx-key,dst=/etc/ssl/nginx/nginx-repo.key,mode=0644 \
+	set -x \
+# Startup script
+	&& chmod +x /deployment/start.sh && touch /.dockerenv \
+# Install prerequisite packages:
+	&& apt-get -y update \
+	&& apt-get -y install apt-transport-https lsb-release ca-certificates wget gnupg2 debian-archive-keyring \
```

```suggestion
	&& apt-get -y install debian-archive-keyring \
```
You've already installed the packages in the other `RUN`, maybe you don't need them there? I'm not sure what the agent script needs.